airbyte-cdk/java/airbyte-cdk/db-sources/src/test/java/io/airbyte/cdk/integrations/debezium/internals/PostgresDebeziumStateUtilTest.java

```diff
@@ -199,47 +197,6 @@ public void LsnCommitTest(final String plugin) throws SQLException {
       Assertions.assertEquals(1, lsnAfterCommit.compareTo(lsnAtTheBeginning));
       Assertions.assertEquals(targetLsn, lsnAfterCommit.asLong());
       Assertions.assertNotEquals(slotStateAtTheBeginning, slotStateAfterCommit);
-
-      // Now check that maybeReplicationStreamIntervalHasRecords behaves as expected.
-
-      final long lsnBeforeBookkeepingStatements = PostgresUtils.getLsn(database).asLong();
-
-      database.execute("SELECT txid_current();");
-      database.execute("CHECKPOINT");
-      final long lsnAfterBookkeepingStatements = PostgresUtils.getLsn(database).asLong();
-      Assertions.assertNotEquals(lsnBeforeBookkeepingStatements, lsnAfterBookkeepingStatements);
-
-      Assertions.assertFalse(postgresDebeziumStateUtil.maybeReplicationStreamIntervalHasRecords(
-          Jsons.jsonNode(databaseConfig),
-          fullReplicationSlot,
-          publication,
-          plugin,
-          lsnBeforeBookkeepingStatements,
-          lsnAfterBookkeepingStatements));
-
-      database.execute("INSERT INTO public.test_table VALUES (3, 'baz');");
-      final long lsnAfterMeaningfulStatement = PostgresUtils.getLsn(database).asLong();
-      Assertions.assertNotEquals(lsnBeforeBookkeepingStatements, lsnAfterMeaningfulStatement);
-
-      Assertions.assertTrue(postgresDebeziumStateUtil.maybeReplicationStreamIntervalHasRecords(
-          Jsons.jsonNode(databaseConfig),
-          fullReplicationSlot,
-          publication,
-          plugin,
-          lsnBeforeBookkeepingStatements,
-          lsnAfterMeaningfulStatement));
-      Assertions.assertTrue(postgresDebeziumStateUtil.maybeReplicationStreamIntervalHasRecords(
-          Jsons.jsonNode(databaseConfig),
-          fullReplicationSlot,
-          publication,
-          plugin,
-          lsnAfterBookkeepingStatements,
-          lsnAfterMeaningfulStatement));
-
-      final var slotStateAtTheEnd = getReplicationSlot(database, fullReplicationSlot, plugin, dbName);
-      Assertions.assertEquals(slotStateAfterCommit, slotStateAtTheEnd);
-
-      container.stop();
```

Comment:
Removed this part of the test case which is what covered the debezium short-circuiting functionality in `maybeReplicationStreamIntervalHasRecords`. It also was flaky which is why the test is no longer disabled.